src/zope/dublincore/tests/test_dcsv.py

```diff
@@ -163,18 +169,21 @@ def test_decode_mixed_values():
     [('a', 'b'), ('c', 'd'), ('', 'e'), ('f', 'g')]
     """
 
+
 def test_decode_duplicate_labels():
     """
     >>> decode('a=b;a=c; a=d')
     [('a', 'b'), ('a', 'c'), ('a', 'd')]
     """
 
+
 def test_encode_empty_list():
     """
     >>> encode([])
```

Comment:
Ok, I see doctests use both encode and decode, so can it be we don't have `doctests = yes` in the flake8 config file?  We should have that.

Reply:
Update: we have that, and the #noqa comment is unnecessary.